### ISSUE TITLE
TK3.19 - Swift Go product page

### ISSUE THREAD

**Issue by mrmjprice** [wmcadigital/wmn-sprints#86]
**Epic:** Introduce the concept of capping

***
#### As a
person who doesn't know what fare capping is

#### I need to
understand what fare capping means

#### So that
I can decide if I want to use fare capping (Swift Go)

***
### Acceptance criteria
- [x] User knows what Swift Go/capping means in plain English
- [x] User can see a list of benefits of using capping
- [x] User can see a list of fare caps
- [x] User knows how to check their capping history
- [x] User can get Swift Go / Activate it on an existing card

***

**Comment by jkmnch** [comment]
Swift Go capping pages and the filtering function have been designed in Figma: https://www.figma.com/file/PDLwmCZgpcrI6hfIHKYLku/New-Swift-Custom-Pages?node-id=0%3A1

Filtering function will need to be developed for this page only. It is likely that the filtering function won't be needed for launch as Swift are still developing their Child and 3-day, 1-week ticket options. But this functionality needs to be built regardless so it's ready for when Swift are ready to offer the new capping options
